README.md

```diff
@@ -12,49 +12,188 @@
 
 ## Usage
 
-### Installing packages
+### Installation
 
 ```sh
-npm install [package-name] --save-dev
+npm install backpack-react-native --save
 ```
 
+<details>
+  <summary>Android</summary>
+
+  #### From source
+
+  Our Android code is written in `Kotlin`, so in order to compile it from source you need to have `org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"` in the `classpath`.
+
```

Comment:
Is it worth adding a command so people can check this? I would have no idea how to check that 😓